File: Akshat Gupta/Day-3B.java
class Solution {
    public boolean isPalindrome(int x) {
        int y=0;
        int temp=x;
        while(x>0){
            int a = x%10;
            y=(y*10)+a;
            x=x/10;
        }
        if(temp==y){
            return true;
        }
        else{
            return false;
        }
    }
}

File: Prafful-Bisht/DAY-3/Day-3-B.cpp
class Solution {
public:
    bool isPalindrome(int x) {
    long long int rev=0;
	int dup=x;
	while(dup>0){
		int lg=dup%10;
		rev = rev*10 + lg;
		dup=dup/10;
	}
    if(x==rev){
        return true;
    }
    else{
        return false;
    }
    }
};